static/app/views/settings/project/projectFilters/projectFiltersSettings.tsx

```diff
@@ -422,9 +436,14 @@ export function ProjectFiltersSettings({project, params, features}: Props) {
                       project.options?.['filters:react-hydration-errors'],
                   }}
                   saveOnBlur
-                  onSubmitSuccess={(
-                    response // This will update our project context
-                  ) => ProjectsStore.onUpdateSuccess(response)}
```

Comment:
these aren't needed?

Reply:
my bad, i was initially trying to put the analytics calls there, but when I removed it, i rm'ed too much. will add back